improvement NXP 29586 property change csv scroller elastic

ataillefer opened the pull request:
- NXP-30564: Replace 10.10 "Test and push" job by a multibranch pipeline
- NXP-30564: Fix MongoDB tests not ignored
- NXP-30564: Remove failing Connect broker readonly test
- NXP-30564: Fix some inconsistent parent POMs
- NXP-30564: Ignore failing ITLayoutDocumentValidationTest ftests
- NXP-30564: Ignore failing RetryPostTest
- NXP-28241: Fix testFatWork by increasing wait duration
- NXP-30564: Don't archive addons ftests artifacts
- NXP-30564: Ignore randomly failing testComplexTopoManyRecordsOnePartition
- NXP-30564: Ignore randomly failing testComplexTopoManyRecordsOneThreadManyPartitions
- NXP-29586: Allow using elastic scroller for csvExport

bdelbosc committed:
NXP-29586: Allow using elastic scroller for csvExport

nuxeojenkins commented:
View issues in JIRA:
- [NXP-29586](https://jira.nuxeo.com/browse/NXP-29586): Add a property to change the CSV Scroller from repo to elastic in 10.10

ataillefer commented:
T&P to double check: https://qa2.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-ataillefer-10.10/69/

ataillefer committed:
NXP-29233: Remove spurious assertion from TestBulkProcessor.testSequentialCommand

This assertion is failing randomly.
Yet, it is to make sure concurrent commands are overlapping during the scrolling, but it's not reliable and not necessary.
The purpose of the test is to ensure that sequential commands do not overlap which is tested in a reliable way just before.
Let's remove the asserttion.

bdelbosc commented on nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/TestBulkProcessor.java:
```diff
@@ -244,10 +244,6 @@ public void testSequentialCommand() throws Exception {
         assertFalse(overlapCommands(
                 results.stream().filter(r -> "dummySequential".equals(r.getAction())).collect(Collectors.toList()),
                 true));
-        // concurrent commands must overlap
```
we are submitting a concurrent command above so it is weird to do nothing with it, we can let a comment like:
// Concurrent commands should overlap but not every time, so it cannot be tested in a reliable way, see NXP-29233
